### DISCUSSION TITLE
Why Figma? When Lunacy would have been better...

### DISCUSSION THREAD

**Discussion by XeonG** [unoplatform/uno#9328] [General]
Just surprised given it's a native desktop app and it's free... what was the reason for not supporting that instead?

https://icons8.com/lunacy   
https://twitter.com/Icons8_Lunacy

**Comment by francoistanguay** [comment]
We started with FIgma because it's the most widely used design tool, and there was a FigmaSharp .net client initiative that looked interesting and it contained everything we needed to support a Design System and code generation: Components, Variants, Component Properties, AutoLayout, Constraints, ...

Our engine is built in a way that we could support other input systems in the future if they are rich enough.

**Comment by XeonG** [comment]
Hopefully you do get something between you and lunacy going as it's much better and faster....  never cared for figma or in browser built stuff ...care even less about it now adobe has acquired it.
